src/full/Agda/TypeChecking/Serialise/Instances/Internal.hs

```diff
@@ -262,14 +264,16 @@ instance EmbPrj NLPSort where
   icod_ (PProp a)   = icodeN 1 PProp a
   icod_ (PInf f a)  = icodeN 2 PInf f a
   icod_ PSizeUniv   = icodeN 3 PSizeUniv
-  icod_ PLockUniv   = icodeN 4 PSizeUniv
+  icod_ PLockUniv   = icodeN 4 PLockUniv
```

Comment:
I made an unrelated change here, I assume this was a typo.

Reply:
Oh, indeed, thanks!